Updates/xxx_c8277_aura.sql

```diff
@@ -0,0 +1,4 @@
+-- Remove aura 13787 from Bloodmaul Soothsayer c.22384
+-- Now handled in ACID since creature addon is not suitable for auras without infinite duration
+UPDATE creature_template_addon SET auras=NULL WHERE entry=22384;
```

Comment:
That creature entry and comment do not match the commit message and Rekk'tilac's entry. Could this be a copy/paste error? Did you forget to save the file?

Reply:
ty